simhash/__init__.py

```diff
@@ -168,6 +168,22 @@ def distance(self, another):
         return ans
 
 
+class MultiSimhash(Simhash):
+    def __init__(self, simhashes):
+        multi_f = 0
+        if not isinstance(simhashes, Iterable):
+            raise Exception('Value passed is not a list of simhashes')
+        for i in simhashes:
+            multi_f = multi_f + i.f
+        if multi_f % 8:
+            raise Exception('Simhashes do not the same length (f)')
+        multi_value = self._concatenate_simhashes(simhashes)
+        super(MultiSimhash, self).__init__(value=multi_value, f=multi_f, hashfunc=simhashes[0].hashfunc)
```

Comment:
Before using `simhases[0]`, we need to check the length of `simhashes` to make sure it's not empty. Also since you are using the first element's `hashfunc`, do we assume all `hashfunc` should be the same for each element in `simahases` list?

Reply:
True, will add some check to look if its empty or not.
Regarding the `hashfunc`, I think it's safe they must be the same. If they are not, which one do you chose for the new multihash?